frontend/app/src/people/main/FocusView.tsx

```diff
@@ -100,6 +101,11 @@ function FocusedView(props: FocusViewProps) {
 
   const isTorSave = canEdit && main.isTorSave();
 
+  const userOrganizations = main.organizations.map((org: Organization) => ({
+    label: org.name[0].toUpperCase() + org.name.slice(1),
```

Comment:
if you could make this a function and write a quick unit test to capitalize that would be good